src/Calendar/Calendar.js

```diff
@@ -397,6 +419,26 @@ class Calendar extends Component {
         return blockedDates[0].isBefore(date, 'day') && blockedDates[1].isAfter(date, 'day');
     }
 
+    isWeekend(date) {
+        return date.day() % 6 === 0;
+    }
+
+    isFocusedDay(date) {
+        return this.state.currentFocusDay.isSame(date);
+    }
```

Comment:
How about we keep a standard style for class functions
1
```js
    isFocusedDay(date) {
        return this.state.currentFocusDay.isSame(date);
    }
```
vs
2
```js
    isFocusedDay = date => {
        return this.state.currentFocusDay.isSame(date);
    }
```
I prefer `#2`